packit_service/worker/handlers/distgit.py

```diff
@@ -349,11 +353,16 @@ def run_for_target(
             logger.debug(f"{self.sync_release_job_type} failed: {ex}")
             # make sure exception message is propagated to the logs
             logging.getLogger("packit").error(str(ex))
+            state = (
+                BaseCommitStatus.neutral
+                if isinstance(ex, ReleaseSkippedPackitException)
+                else BaseCommitStatus.failure
+            )
             # eat the exception and continue with the execution
             self.sync_release_helper.report_status_for_branch(
                 branch=branch,
                 description=f"{self.job_name_for_reporting.capitalize()} failed: {ex}",
-                state=BaseCommitStatus.failure,
+                state=state,
                 url=url,
             )
             model.set_status(status=SyncReleaseTargetStatus.error)
```

Comment:
Do you think there is an easy way to show the skipped run on the Dashboard?

Reply:
For the dashboard I don't see any other way than to add a new value in `SyncReleaseTargetStatus`. Let me know if I do it.